.travis.yml

```diff
@@ -0,0 +1,11 @@
+language: python
+
+install:
+  - pip install -r requirements.txt
+  - git clone https://github.com/simphony/JYU-LB.git
```

Comment:
It was not clear that it was a depedency it should be moved to the requirements.txt file